ext/ext.go

```diff
@@ -0,0 +1,160 @@
+package ext
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"runtime/debug"
+	"sort"
+	"strings"
+	"sync"
+)
+
+// TODO: Make an ExtensionRegistry?
+//
+//nolint:gochecknoglobals
+var (
+	mx         sync.RWMutex
+	extensions = make(map[ExtensionType]map[string]*Extension)
+)
+
+// ExtensionType is the type of all supported k6 extensions.
+type ExtensionType uint8
+
+// All supported k6 extension types.
+const (
+	JSExtension ExtensionType = iota + 1
```

Comment:
That looks like an opportunity to learn something: why the `+1` on the `iota` here? 😇

Reply:
Just a convention to avoid constants with a value of 0. That way you can identify when a value was set or not.